test/Garnet.test/CredentialManager.cs

```diff
@@ -35,9 +35,36 @@ public void GenerateCredentials(string TestFolder, ServerCredential[] customCred
 
                 var user = cred.user;
                 var password = cred.IsClearText ? cred.password : Convert.ToHexString(cred.hash);
-                aclConfig += $"user {user} on " +
-                    $"{(cred.IsClearText ? ">" : "#")}{password} " +
-                    $"{(cred.IsAdmin ? "+@admin" : "-@admin")}";
+
+                if (user == "default")
+                {
+                    aclConfig += "user default on ";
+                    if (cred.IsClearText)
+                    {
+                        aclConfig += ">";
+                    }
+                    else
+                    {
+                        aclConfig += "#";
+                    }
+                    aclConfig += password;
+
+                    // default user ACL list is: ~* &* +@all
+                    // but we don't support key or channel filters yet
+                    aclConfig += " +@all";
+                }
+                else
+                {
+                    aclConfig += $"user {user} on " +
+                    $"{(cred.IsClearText ? ">" : "#")}{password} ";
+
+                    aclConfig += "+@all";
+
+                    if (!cred.IsAdmin)
+                    {
+                        aclConfig += " -@admin";
+                    }
+                }
```

Comment:
Good catch, this is a hold-over from moving from the old implementation to the new one where the default user was still under-implemented.  This doesn't affect correctness (until/unless key or channel filters are implemented), but is pointless code since the default user (if not otherwise configured) gets this exact ACL anyway.